weldx/visualization/matplotlib_impl.py

```diff
@@ -173,7 +173,7 @@ def plot_local_coordinate_system_matplotlib(
     label : str
         Name of the coordinate system
     time : pandas.DatetimeIndex, pandas.TimedeltaIndex, List[pandas.Timestamp], or \
-           LocalCoordinateSystem
+           ~weldx.transformations.LocalCoordinateSystem
```

Comment:
@vhirtham @marscher 
for classes that are ambiguous (because defined in different init namespaces or other reasons) this is the explicit way to link in docstrings
For type hints this is not necessary (since they get resolved from the file imports I guess)

using the `~` will shorten the visible link in the documentation to the final classifyer, see https://stackoverflow.com/a/21297186/11242411

Reply:
That looks nice. Thanks a lot for taking a shot on this. Sphinx is sometimes hard to convince to do the simplest things.

Reply:
Ain't the convention to omit the types in the docstring, but let it be derived from the type annotation?

Reply:
yes, if possible types can and should be inferred from the type hints but can be set manually like here. I have left this example in the docstrings since I think in this case it looks nicer than `Union[pandas.DatetimeIndex, pandas.TimedeltaIndex, List[pandas.Timestamp], LocalCoordinateSystem]`

Reply:
This union format issue is solved by the software. I've seen an option to format it without the Union thing, but can not recall it right now. I think we should stick to the type annotation wherever possible to avoid duplication.